src/components/extensive/Modal/ModalWithLogo.tsx

```diff
@@ -63,8 +91,8 @@ const ModalWithLogo: FC<ModalWithLogoProps> = ({
             style={{ minHeight: iconProps?.height ?? iconProps?.width ?? 40 }}
           />
         </When>
-        <div className="flex items-center justify-between">
-          <Text variant="text-24-bold">{title}</Text>
+        <div className="mb-8 flex items-center justify-between">
+          <Text variant="text-24-bold">{title + "XD"}</Text>
```

Comment:
This is a little odd - why is this being added to the title? Does it need to be translated? Or is it 🛅 ?

Reply:
I used to debug, seems that I have forgot to delete.
Is deleted now.